jablib/src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2945,3 +2945,8 @@ The\ following\ providers\ are\ available\:=The following providers are availabl
 Unable\ to\ open\ file\ '%0'.=Unable to open file '%0'.
 Unknown\ export\ format\ '%0'.=Unknown export format '%0'.
 Updating\ PDF\ metadata.=Updating PDF metadata.
+'%0'\ file\ already\ exists\!\ Use\ -f\ or\ --force\ to\ overwrite.='%0' file already exists! Use -f or --force to overwrite.
```

Comment:
Use . instead of exclamation mark (!). We use ! very seldomly. The fact that a file already exists can be expected by the user. Thus no need for an extra emphasis.